### PR TITLE
Update troubleshooting doc for class not found error

### PR DESCRIPTION
**Pull request by testforstephen**
Signed-off-by: Jinbo Wang <jinbwan@microsoft.com>

### CODE REVIEW

File: Troubleshooting.md [hunk 1]
```diff
@@ -65,43 +76,41 @@ There are two possible reasons for this error.
...
-When the `mainClass` is unconfigured in the *launch.json*, the debugger will resolve a class with main method automatically. This error indicates the debugger doesn't find any main class in the whole workspace.
-
-### Try:
-1. Check at least one main class exists in your workspace.
-2. If no main class exists, please create a main class first. Otherwise, it's probably because the language server fails to start. Please reference the [language server troubleshooting](#try) paragraph for more troubleshooting info.
+Below are the common failure reasons.
+- 'C:\demo\org\microsoft\app\Main.java' is not a valid class name.
+- Main class 'org.microsoft.app.Main' doesn't exist in the workspace.
+- Main class 'org.microsoft.app.Main' isn't unique in the workspace.
+- The project 'demo' is not a valid java project.
```
Comment (jdneo) [inline Troubleshooting.md]: Is there anyway we can let the users know what is a valid java project?

File: Troubleshooting.md [hunk 2]
```diff
@@ -65,43 +76,41 @@ There are two possible reasons for this error.
...
-## Cannot find a class with the main method
+## Failed to resolve classpath:
 ### Reason:
-When the `mainClass` is unconfigured in the *launch.json*, the debugger will resolve a class with main method automatically. This error indicates the debugger doesn't find any main class in the whole workspace.
-
-### Try:
-1. Check at least one main class exists in your workspace.
-2. If no main class exists, please create a main class first. Otherwise, it's probably because the language server fails to start. Please reference the [language server troubleshooting](#try) paragraph for more troubleshooting info.
+Below are the common failure reasons.
+- 'C:\demo\org\microsoft\app\Main.java' is not a valid class name.
```
Comment (jdneo) [inline Troubleshooting.md]: `org` -> `com`?

File: Troubleshooting.md (hunk 1 above)
Comment (testforstephen) [reply Troubleshooting.md]: yes, the debugger will provide fix proposals quick pick for user to correct it if it finds the project is not valid.

Comment (jdneo) [reply Troubleshooting.md]: Upper case the word `Java`.

**Pull request merged**